package.json

```diff
@@ -23,6 +23,7 @@
   "homepage": "https://github.com/CuongNgMan/Mentoring-HTML-CSS#readme",
   "devDependencies": {
     "@babel/core": "^7.16.0",
+    "@babel/node": "^7.16.0",
     "@babel/preset-env": "^7.16.4",
     "babel-jest": "^27.3.1",
```

Comment:
helmet packages has not been added into devDependecies :) 
I had to manually install it to run tests. 
Please, don't forget adding all necessary dependencies into package.json